### PR TITLE
Update for 4.26

### PR DESCRIPTION
**Pull request by lboyd93**
This PR updates the changelog and readme for 4.26.

@annelfitz will you review please? Thanks!

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -193,43 +29,11 @@ The following classes, methods, properties and events have been deprecated for a
...
-- When embedding a video in a popup using the HTML `<video>` tag, the video will automatically resize to fit inside the popup if it is larger than the popup window.
+- TBD
 
 ## Deprecations
 
-The following are deprecated and will be removed in a future release. For anything deprecated in 4.24 and earlier, additional information and links are in the [release notes](https://developers.arcgis.com/javascript/latest/release-notes/#deprecated-classes-properties-methods-events).
+The following are deprecated and will be removed in a future release. For anything deprecated in 4.25 and earlier, additional information and links are in the [release notes](https://developers.arcgis.com/javascript/latest/release-notes/#deprecated-classes-properties-methods-events).
 
 <details>
   <summary>Click to expand!</summary>
```
Comment (annelfitz) [inline CHANGELOG.md:39]: should we update the list of deprecations too? or are there no changes here yet

**Review by annelfitz — COMMENTED** [review]
LGTM!! 🚀

File: CHANGELOG.md (hunk 1 above)
Comment (lboyd93) [reply CHANGELOG.md:39]: I can update what's on here with what's on the release notes for 4.25. Is that what we typically do?

Comment (lboyd93) [reply CHANGELOG.md:39]: Thanks @annelfitz ! Updated the list.

**Comment by lboyd93** [comment]
@bsvensson - this is ready to merge!

**Comment by andygup** [comment]
Ditto, LGTM!

**Pull request merged**